grails-app/controllers/com/mini/asaas/payment/PaymentController.groovy

```diff
@@ -83,4 +83,25 @@ class PaymentController {
             redirect(view: "edit", params: params)
         }
     }
+
+    def showToPayer() {
```

Comment:
O que vcs acham de separar estes dois métodos que são ações do pagador em outro controller? Pois depois que for implementado a gestão de acessos, ficará mais simples.